docs/dg/dev/architecture/technology-stack.md

```diff
@@ -25,9 +25,7 @@ Spryker uses a set of well-known tools:
 * *Redis*. Redis is the most popular key-value database; the name is an abbreviation for Remote Dictionary Server. In Spryker it's used as a client-side data source for localized content. The Redis key-value database avoids the necessity of making queries to the SQL database, which can come with a high cost. The data stored in Redis is kept in sync with the data stored in the SQL database through specialized cronjobs. Redis supports replication, so it's a solution that enables scalability.
 * *Elasticsearch*. Elasticsearch is a distributed search engine that offers an easy-to-configure and easy-to-integrate solution for making searches fast. The relevant search data is stored in dedicated storage, so is similar to using Redis storage; it avoids making a costly query to the SQL database. The data stored in Elasticsearch is updated through cronjobs.
 * *Symfony*. Symfony is the leading PHP framework for creating MVC web applications. Spryker implements the model-view-controller design pattern with Symfony. MVC design pattern aims to separate the business logic from the view, making the source code easier to understand and maintain due to this separation.
-* *Silex (deprecated)*. Silex is a PHP micro-framework based on Symfony components; it offers Twig—a fast and modern PHP templating engine.
```

Comment:
but we still have a lot of "Silex" related classes in the master branch https://github.com/search?q=repo%3Aspryker%2Fspryker%20silex&type=code
it is deprecated but available

Reply:
That's good, it will also be forever in the repo history. 
But Silex is not a part of our current stack.